src/platform/packages/shared/kbn-visualization-utils/src/get_timezone.test.ts

```diff
@@ -33,6 +33,15 @@ describe('getTimeZone', () => {
     ).toEqual('America/New_York');
   });
 
+  it('returns local time zone when uiSettings returns Browser as non default', () => {
+    expect(
+      getTimeZone({
+        get: () => 'Browser',
+        isDefault: () => false,
```

Comment:
Would be great to update tests now since we don't call `isDefault` anymore.

Reply:
fixed on [`0af1c9c` (#220658)](https://github.com/elastic/kibana/pull/220658/commits/0af1c9c0d532c93f87ae0c7f90c1ea6273743d06)